### ISSUE TITLE
development not normal but production is norlmal when i use element3 with iconfont.

### ISSUE THREAD

**Issue by mrandpz** [cawa-93/vite-electron-builder#49]
![image](https://user-images.githubusercontent.com/20697372/104543867-71715000-5661-11eb-9c3b-9b5467214da2.png)

**Comment by cawa-93** [comment]
I have an assumption that you are trying to use the fs module. More I can't say anything without knowing the details, code and how to reproduce this problem.

**Comment by mrandpz** [comment]
- git clone https://github.com/cawa-93/vite-electron-builder.git
- yarn
- yarn add element3
- index.ts
```js
import "element3/lib/theme-chalk/index.css";
import { createApp } from "vue";
import Element3 from "element3";
import App from "./App.vue";
import router from "/@/router";

createApp(App).use(Element3).use(router).mount("#app");
```
- App.vue  add 
```
<i class="el-icon-edit"></i>
```

Then there will be font resource 404, by the way, how to configure external, such as vue. I configured it in the external parameter of renderer.vite.js and it did not take effect

**Comment by cawa-93** [comment]
Fix 00fcd25514d981b6de5fa3cb05f250e0d8aec54d
